lib/gui/pages/finish/templates/success.tpl.html

```diff
@@ -1,35 +1,45 @@
 <div class="page-finish row around-xs">
   <div class="col-xs">
-    <div class="box text-center">
-      <h3 class="title"><span class="tick tick--success space-right-tiny"></span> Flash Complete!</h3>
-      <p class="soft space-vertical-small" ng-show="finish.settings.get('unmountOnSuccess')">Safely ejected and ready for use</p>
+    <div class="box center">
+      <div class="col-xs-5">
+        <h3 class="title"><span class="tick tick--success space-right-tiny"></span> Flash Complete!</h3>
+      </div>
 
-      <div class="row center-xs space-vertical-medium">
-        <div class="col-xs-4 space-medium">
-          <div class="box">
-            <p class="soft button-label">Would you like to flash the same image?</p>
+      <div class="col-xs-7">
+        <button class="button button-primary button-brick" ng-click="finish.restart()">
+          <b>Flash Another</b>
+        </button>
 
-            <button class="button button-primary button-brick" ng-click="finish.restart({ preserveImage: true })">
-              Use <b>same</b> image
-            </button>
-          </div>
-        </div>
+        <button class="button button-secondary button-brick" ng-click="finish.restart({ preserveImage: true })">
+          Show Details
+        </button>
+      </div>
+    </div>
 
-        <div class="col-xs separator-xs"></div>
+    <div class="box center">
+      <safe-webview src="'https://etcher.io/'"
+        ng-show="finish.shouldShowWebview"
+        timeout="2000"
+        callback="finish.setShowWebview"></safe-webview>
```

Comment:
Can we move the logic that makes the `safe-webview` only appear if it has fully loaded to the component itself? The directive can then simply take an attribute that points to a boolean variable that will be set once its finally visible. For example:

```html
<safe-webview src="'https://etcher.io/'" timeout="2000" visible="myVisiblePropertyWhichIsInitiallySetToFalse"></safe-webview>

div class="fallback-banner" ng-hide="myVisiblePropertyWhichIsInitiallySetToFalse">
```